app/assets/javascripts/abstract-list.js

```diff
@@ -96,9 +97,23 @@ require(["lib/models", "lib/tools", "knockout", "sammy", "lib/offline"], functio
             }
             self.selectedAbstract(abstract);
             document.title = abstract.title; //FIXME add conference
+            //if user is not logged in
+            if(loggedIn.indexOf('true')>=0){
```

Comment:
The `loggedIn` parameter is treated as a string. Could it be a boolean instead so we don't have to do a substring search? Where does it come from?